frontend/src/components/ui/ListSkeleton.tsx

```diff
@@ -1,8 +1,8 @@
 import React from "react";
```

Comment:
To be honest, I don't think it's such a good idea in doing that change regarding refactoring the skeletons in this pr. I would prefer if just **undo** this change. We can however discuss in another issue / ticket how we would refactor it. Also I would like to adjust all the skeletons more to the  actual content. Sorry for your extra work.

Reply:
> Also I would like to adjust all the skeletons more to the actual content. Sorry for your extra work.

That's all right. Apart from creating a separate copy of the skeleton component, do I need to modify anything in the skeleton for the spells page?

Reply:
nope, you shouldn't have to change anything, except if it were completely offset, but I doubt it